### ISSUE TITLE
Option to hide note properties (e.g. tags) in the teleprompter

### ISSUE THREAD

**Issue by orkunzozturk** [lumetrium/obsidian-teleprompter#14]
I organize my notes with custom properties such as "Campaign" and the standard tag property.
I couldn't find an option in the Teleprompter plugin to hide page properties. No matter what the option for displaying note properties are in Settings -- Editor -- Properties in Document, Teleprompter displays them as "Visible" like below:

<img width="631" alt="image" src="https://github.com/lumetrium/obsidian-teleprompter/assets/80598196/07cb84ad-871b-4b14-8496-c8a143b36d88">

**Comment by lumetrium** [comment]
Good point. There should definitely be an option for this, or at the very least, it should respect the editor's settings. I'll think how to better implement this.

**Comment by lumetrium** [comment]
Resolved in the latest 1.4.0 release (details here: https://github.com/lumetrium/obsidian-teleprompter/releases/tag/1.4.0), the teleprompter now follows the settings in Obsidian under "Settings" -> "Editor" -> "Properties in document". If the properties are set to "Hidden", they will not appear in the teleprompter window.

In the future, this will likely become a separate option within the teleprompter settings. I've added this to the backlog.
